.github/workflows/bump-version.yml

```
@@ -16,20 +16,35 @@ jobs:
     env:
       MISE_ENV: ci
     steps:
+      - uses: actions/create-github-app-token@v3
```

actionlint (1.7.12)

[error] missing input "app-id" which is required by action "actions/create-github-app-token@v3". all required inputs are "app-id", "private-key"

(action)